src/main/java/com/orange/main/price/bo/Price.java

```diff
@@ -0,0 +1,95 @@
+package com.orange.main.price.bo;
+
+import java.math.BigDecimal;
+import java.util.TimeZone;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Table(name = "price")
+public class Price {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    private int productId;
+    public int getProductId() {
+        return productId;
+    }
+
+    public void setProductId(int productId) {
+        this.productId = productId;
+    }
+
+
+
+    private BigDecimal price;
+    private BigDecimal discount;
+    private String source_url;
```

Comment:
IIUC, you should no align the field name to the DB column name. On the contrary, keeping the JAVA naming style.